client/beacon-client/service.go

```diff
@@ -0,0 +1,208 @@
+package beacon
```

Comment:
Please match package names with the file path. 

Change this to beaconclient or rename the directory to beacon. Also you might want to rename the directory to beaconclient without the -. Otherwise they have to alias the import like you are doing elsewhere.